cli/dcoscli/cluster/main.py

```diff
@@ -147,10 +148,22 @@ def _list(json_, attached, linked=False):
     if attached:
         clusters = [c.dict() for c in cluster.get_clusters()
                     if c.is_attached()]
-    elif linked:
-        clusters = [c.dict() for c in cluster.get_linked_clusters()]
     else:
-        clusters = [c.dict() for c in cluster.get_clusters(True)]
+        if linked:
+            clusters = cluster.get_linked_clusters()
+        else:
+            clusters = cluster.get_clusters(True)
+
+        nb_clusters = len(clusters)
+
+        if nb_clusters > 1:
```

Comment:
It would probably be cleaner in the code to perform the requests concurrently no matter the number of clusters

Reply:
updated